lib/elixir/lib/access.ex

```diff
@@ -19,26 +20,28 @@ defprotocol Access do
       iex> map[:a]
       1
 
-      iex> star_ratings = %{1.0 => "★", 1.5 => "★☆", 2.0 => "★★"}
+      iex> star_ratings = %{1.0 => "★", 1.5 => "★½", 2.0 => "★★"}
       iex> star_ratings[1.5]
-      "★☆"
+      "★½"
 
   The key comparison must be implemented using the `===` operator.
   """
 
   @doc """
-  Accesses the given key in the container.
+  Returns the value of the given key in the container.
```

Comment:
I think we should avoid starting functions with "returns". The huge majority of them return something. I think the previous description was just fine. :)